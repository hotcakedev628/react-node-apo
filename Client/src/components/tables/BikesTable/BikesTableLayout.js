import React from 'react'


export const BikesTableLayout = ({ TableHeaders, TableBodyContent }) => {
    return (
        <div className="mt-4">
            {/* <h3 className="panel-title">
            <NavLink title="Add New Bike" id="addButton" className="text-info pull-right fa fa-plus link" to='/addBike' aria-hidden="true"></NavLink>
        </h3> */}
            <table id="BikesTable" className="table table-hover">
                <thead>
                    <tr>
                        <TableHeaders />
                    </tr>
                </thead>
                <tbody>
                    <TableBodyContent />
                </tbody>
            </table>
        </div>

    )
}

