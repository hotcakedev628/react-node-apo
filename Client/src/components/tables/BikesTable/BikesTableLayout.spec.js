import renderer from 'react-test-renderer';
import {BikesTableLayout} from './BikesTableLayout';
import React from 'react'



describe('BikesTableLayout', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<BikesTableLayout 
        TableHeaders={()=>(<li>HH</li>)}
        TableBodyContent={()=>(<li>C</li>)}
        />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
