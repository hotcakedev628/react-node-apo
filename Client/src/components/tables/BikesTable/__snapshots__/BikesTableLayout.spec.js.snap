// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BikesTableLayout should match snapshot 1`] = `
<div
  className="mt-4"
>
  <table
    className="table table-hover"
    id="BikesTable"
  >
    <thead>
      <tr>
        <li>
          HH
        </li>
      </tr>
    </thead>
    <tbody>
      <li>
        C
      </li>
    </tbody>
  </table>
</div>
`;
